test/github_pullrequest_test.go

```diff
@@ -135,6 +137,57 @@ func TestGithubPullRequestSecondBadYaml(t *testing.T) {
 	golden.Assert(t, res.CheckRuns[0].GetOutput().GetText(), strings.ReplaceAll(fmt.Sprintf("%s.golden", t.Name()), "/", "-"))
 }
 
+// TestGithubPullRequestInvalidSpecValues tests invalid field values of a PipelinRun and
+// ensures that these validation errors are reported on UI.
+func TestGithubPullRequestInvalidSpecValues(t *testing.T) {
```

Comment:
@zakisk lets not run all tests on every PR otherwise we might hit a rate limit
Please add this condition
```
	if os.Getenv("NIGHTLY_E2E_TEST") != "true" {
		t.Skip("Skipping test since only enabled for nightly")
	}
```
which will just run on nightly 
ref : https://github.com/openshift-pipelines/pipelines-as-code/blob/main/test/github_pullrequest_privaterepository_test.go#L20-L22

Reply:
is this test only for nightly?

Reply:
Yes